ui/pages/onboarding-flow/privacy-settings/setting.js

```diff
@@ -33,7 +33,9 @@ export const Setting = ({
         <Text variant={TextVariant.bodyLgMedium} fontWeight={FontWeight.Bold}>
           {title}
         </Text>
-        <Text variant={TextVariant.bodyMd}>{description}</Text>
+        <Text variant={TextVariant.bodyMd} as="div">
+          {description}
+        </Text>
```

Comment:
Changing root element to div so there are no semantic errors

### Before

![Screenshot 2023-08-14 at 1 51 07 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/4b8a2a32-948e-4653-ab11-442e5921be5b)

### After

![Screenshot 2023-08-14 at 1 51 49 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/dcdcd3ae-8c0e-41da-99e6-ff09b18b9c61)